src/main/resources/db/migration/metadata/V1__add_metaprotocols_table.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TYPE idl AS ENUM ('avro', 'protobuf', 'mu', 'openapi', 'scala');
+CREATE TYPE idl AS ENUM ('avro', 'protobuf', 'mu', 'openapiyaml', 'openapijson', 'scala');
```

Comment:
We shouldn't change migrations. If we need a change we need to create a new migration file. The idea with the migrations is to manage DB evolutions and changing a previous one makes for an existing installation would make it fail.

I think [this post](https://blog.yo1.dog/updating-enum-values-in-postgresql-the-safe-and-easy-way/) covers what we need here, specially in the `Updating/Renaming a Value`